src/main/scala/com/adobe/api/platform/runtime/metrics/EventConsumer.scala

```diff
@@ -32,6 +34,23 @@ case class EventConsumer(settings: ConsumerSettings[String, String], recorders:
   materializer: ActorMaterializer) {
   import EventConsumer._
 
+  //Record the rate of events received
+  private val activationCounter = Kamon.counter("openwhisk.userevents.global.activations")
+  private val metricCounter = Kamon.counter("openwhisk.userevents.global.metric")
+
+  private val statusCounter = Kamon.counter("openwhisk.userevents.global.status")
+  private val coldStartCounter = Kamon.counter("openwhisk.userevents.global.coldStarts")
+
+  private val statusSuccess = statusCounter.refine("status" -> Activation.statusSuccess)
+  private val statusFailure = statusCounter.refine("status" -> "failure")
+  private val statusApplicationError = statusCounter.refine("status" -> Activation.statusApplicationError)
+  private val statusDeveloperError = statusCounter.refine("status" -> Activation.statusDeveloperError)
+  private val statusInternalError = statusCounter.refine("status" -> Activation.statusInternalError)
+
+  private val waitTime = Kamon.histogram("openwhisk.userevents.global.waitTime", MeasurementUnit.time.milliseconds)
+  private val initTime = Kamon.histogram("openwhisk.userevents.global.initTime", MeasurementUnit.time.milliseconds)
+  private val duration = Kamon.histogram("openwhisk.userevents.global.duration", MeasurementUnit.time.milliseconds)
```

Comment:
Should we have a `gauge` for `memory` too? This would help compare the utilization with the sum of memory configuration for all invocations.

Reply:
Would look into this in next PR. Need to understand the intention better